src/styles/book.css

```diff
@@ -0,0 +1,219 @@
+body {
+  background-color: #efefef;
+}
+
+#main-container {
+  display: flex;
+  flex-direction: column;
+  padding: 15px 77px;
+}
+
+.display-books {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  border: 1px solid #e8e8e8;
+  padding: 20px;
+  margin-bottom: 20px;
+  background-color: white;
+}
+
+#nav-container {
+  width: 100%;
+  display: flex;
+  align-items: center;
+  border-bottom: 1px solid #e8e8e8;
+  background-color: white;
+}
+
+#nav-container h1 {
+  color: #0290ff;
+  font-size: 1.875rem;
+  margin-left: 5rem;
+}
+
+#pages-container {
+  display: flex;
+  align-items: center;
+}
+
+#pages-container li {
+  list-style-type: none;
+  padding-left: 20px;
+  padding-right: 20px;
+  letter-spacing: 1px;
+}
+
+.user-icon {
+  border: 1px solid black;
+  padding: 13px;
+  border-radius: 24rem;
+  color: #0290ff;
+  margin-left: 77rem;
+}
+
+.Books {
+  color: #121212;
+}
+
+.Categories {
+  color: #a3a3a3;
+}
+
+#all-btn {
+  margin-left: -5px;
+  display: flex;
+  gap: 1rem;
+  align-items: center;
+}
```

Comment:
- [ ] The spacing between each button seems perfectly balanced but the separator `|` is distracting the spacing so here are my suggestions:

1. Remove the `border-right` styling from the class `all-btn .comments` and `#all-btn .remove` 👍🏼 
2. Add `<span>|</span>` between each button in HTML file 👍🏼 

### Your Design
![Screenshot from 2023-05-20 14-14-51](https://github.com/Shaheryar0054/bookstore-using-react/assets/68030297/06e1068b-40a4-46f2-b617-c22b7abbe5ca)

### System Design
![Screenshot from 2023-05-20 14-15-22](https://github.com/Shaheryar0054/bookstore-using-react/assets/68030297/3191b871-512b-4955-9f68-16e32160abaf)